pom.xml

```diff
@@ -7,8 +7,23 @@
   <version>1.14-SNAPSHOT</version>
   <packaging>maven-plugin</packaging>
 
-  <name>maven-license-plugin Maven Plugin</name>
-  <url>http://maven.apache.org</url>
+  <name>License Maven Plugin</name>
+  <url>https://github.com/cloudbees/maven-license-plugin</url>
+  <description>Maven plugin to process license information.</description>
+  <licenses>
+    <license>
+      <name>Apache 2.0 License</name>
```

Comment:
Parent actually indicated MIT, but this contradicts the sole file with a license header https://github.com/cloudbees/maven-license-plugin/blob/6ceec8584a32967cfe8828126799e9d7f16e4461/src/main/java/com/cloudbees/maven/license/ProcessMojo.java#L3-L17